messages/verify.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "for an npm package validate the associated digital signature if it exits",
```

Comment:
This might have been what it was before but it reads really weird to me. Can we say `validate a digital signature for a npm package`?